src/utils/style-formatter.js

```diff
@@ -67,9 +68,10 @@ export default {
       const { width, radius } = style.border;
       const { offsetHeight, offsetWidth } = element;
       const lengthShortSide = offsetHeight < offsetWidth ? offsetHeight : offsetWidth;
+      const defaultBorderColor = colorUtils.lightenOrDarkenColor(backgroundColor, -0.15);
```

Comment:
Since we are not going to use this anywhere else (for now), can be a constant inside the `lightenOrDarkenColor`.